go.mod

```diff
@@ -16,3 +16,45 @@ require (
 	k8s.io/client-go v0.21.2
 	k8s.io/klog/v2 v2.8.0
 )
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
Maybe a 
BEGIN Indirect
END Indirect
would be better than the comment on all the dependencies.

Reply:
This file is auto generated by `go mod tidy`. It is not supposed to be modified by hand.